backend/scrapers/em2.js

```diff
@@ -0,0 +1,371 @@
+    this.talkToDB({
+      type: 'db-request',
+      subtype: 'saveAllowedTitle',
+      messageData: {
+        string: event.title,
+        reason: reasons.reverse().join(', '),
+      },
+    });
+    return goodTermsRes;
+  }
+
+
+  this.talkToDB({
+    type: 'db-request',
+    subtype: 'saveRefusedTitle',
+    messageData: {
+      string: event.title,
+      reason: reasons.reverse().join(', '),
+    },
+  });   
+  return {
+    event,
+    reason: reasons.reverse().join(', '),
+    success: false,
+  };
+};
+// #endregion                          MAIN PAGE EVENT CHECK
+
+// #region [rgba(0, 180, 0, 0.1)]      SINGLE PAGE EVENT CHECK
+// #endregion                          SINGLE PAGE EVENT CHECK
+
+// #region [rgba(0, 240, 0, 0.1)]      MAIN PAGE
+scraper.mainPage = async function () {
+  const availableBaseEvents = await this.checkBaseEventAvailable(workerData.family);
+  if (availableBaseEvents) {
+    const thisWorkersEvents = availableBaseEvents.filter(
+      (eventEl, index) => index % workerData.workerCount === workerData.index,
+    );
+    return this.mainPageEnd({ stopFunctie: null, rawEvents: thisWorkersEvents });
+  }
+
+  const { stopFunctie, page } = await this.mainPageStart();
+
+  await this.waitTime(50);
+
+  const cookiesNodig = await page.evaluate(() =>
+    document.querySelector('html').classList.contains('has-open-cookie'),
+  );
+
+  if (cookiesNodig) {
+    await page.evaluate(() => {
+      document.querySelector("[name*='marketing']").click();
+      document.querySelector('.cookie__settings .cookie__process').click();
+    });
+    await this.waitTime(50);
+  }
+
+  await this.autoScroll(page);
+  await this.autoScroll(page);
+
+  let rawEvents = await page.evaluate(
+    // eslint-disable-next-line no-shadow
+    ({ workerData, unavailabiltyTerms }) =>
+      Array.from(document.querySelectorAll('.agenda-container .agenda-item'))
+        .filter((eventEl) => !eventEl.classList.contains('is-hidden'))
+        .map((eventEl) => {
+          const title = eventEl.querySelector('h3')?.textContent ?? '';
+          const res = {
+            anker: `<a class='page-info' href="${document.location.href}">${workerData.family} - main - ${title}</a>`,
+            errors: [],
+            title,
+          };
+
+          try {
+            const DDslashMM = eventEl.querySelector('.agenda-item-date')?.textContent ?? null;
+            const DD = DDslashMM.split("/")[0].trim();
+            const MM = DDslashMM.split("/")[1].trim();
+            res.startDate = `2024-${MM}-${DD}`;
+          } catch (caughtError) {
+            res.errors.push({
+              error: caughtError,
+              remarks: `date time faal ${title}.`,
+            });
+          }
+          res.shortText = eventEl.querySelector('.agenda-item-content p')?.textContent ?? '';
+          res.venueEventUrl = eventEl.querySelector('.more-button')?.href ?? null;
+
+          // TODO: Hoe te bepalen of show is uitverkocht?
```

Comment:
![afbeelding](https://github.com/sjerpbouwtsites/rockers-concertagenda/assets/34159539/cb957927-545a-48df-9bdf-75258e5f7850)

Reply:
in .event-value op single